### PR TITLE
#597 - Change loading to indefinite progress bar for Inspector Launcher

### PR DESCRIPTION
**Pull request by Googleton**
## Description

- [x] Change loading from spinner to indefinite progress bar
- [x] Migrate InspectorLauncher.vue to Composition API

## Related Issue

#432 => InspectorLauncher.vue migrated

### CODE REVIEW

File: frontend/src/views/main/project/InspectorLauncher.vue [hunk 1]
```diff
@@ -16,6 +16,15 @@
         </v-radio-group>
       </div>
       <div v-else>No dataset uploaded yet on this project.</div>
+      <v-spacer/>
+      <div v-if="creatingInspection">
+        <span>Scoring the entire dataset. This operation can take a while...</span>
```
Comment (alexcombessie) [inline frontend/src/views/main/project/InspectorLauncher.vue:21]: Would just need to add "Scoring the entire dataset of {fileSizeInHumanReadableFormat}"

**Review by alexcombessie — CHANGES_REQUESTED** [review]

**Review by alexcombessie — COMMENTED** [review]
Thanks @Googleton ! I verified that it works well functionally.

On the code part, I see several changes in the TypeScript part that are not directly related to the issue. Is it pure refactoring or have you changed the actual logic of functions?

Also, what is the function of the vue2-vue3-migration.md file in this PR?

**Comment by Googleton** [comment]
Those TypeScript changes are related to #432 which is a migration from Vue 2's Options API to Vue 3's Composition API
The general rule of thumb, if we modify a Vue 2 component, migrate it to Vue 3 in the same PR so the migration progresses without dedicating a larger amount of time to do it.

The documentation file is here as a cheatsheet to easily find common patterns in that migration

**Comment by alexcombessie** [comment]
@Googleton okay thanks for the explanation. In that case I will let @andreybavt lead the code review of the Vue3 migration.
On the functional side, the PR is okay for me. Thanks for your good work!

PS: merry Christmas in delay 🎄

**Review by andreybavt — APPROVED** [review]

**Comment by rabah-khalek** [comment]
I really like this bar addition, can we just find an alternative to `NaN`?
Screenshot 2023-01-11 at 11 23 27

**Pull request merged**